src/main/default/lwc/lookup/lookup.js

```diff
@@ -112,16 +116,13 @@ export default class Lookup extends LightningElement {
 
         // Compare clean new search term with current one and abort if identical
         const newCleanSearchTerm = newSearchTerm.trim().replace(/\*/g, '').toLowerCase();
```

Comment:
do not remove this. instead pass in an api boolean, wether you want to execute the search for empty values too, so the default behaviour does not change